src/statement-list/converters/expr2statement_list.cpp

```diff
@@ -25,3 +65,219 @@ std::string type2stl(const typet &type, const namespacet &ns)
   // CBMC types to STL code.
   return type2c(type, ns);
 }
+
+expr2stlt::expr2stlt(const namespacet &ns)
+  : inside_bit_string(false), is_reference(false), ns(ns)
+{
+}
+
+std::string expr2stlt::convert(const exprt &expr)
+{
+  if(ID_and == expr.id())
+    convert_and(expr);
+  else if(ID_or == expr.id())
+    convert_or(expr);
+  else if(ID_xor == expr.id() || ID_notequal == expr.id())
+    convert_xor(expr);
+  else if(ID_equal == expr.id())
+    convert_equal(expr);
+  else if(ID_symbol == expr.id())
+    convert_symbol(expr);
+  else if(ID_not == expr.id())
+    convert_not(expr);
+  else // TODO: support more instructions in expr2stl.
+    return expr2c(expr, ns);
+
+  return result.str();
+}
+
+void expr2stlt::convert_and(const exprt &expr)
```

Comment:
The following few functions follow the same pattern just with a different operator string literal. Extract into static function.